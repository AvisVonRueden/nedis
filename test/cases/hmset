*6
$5
hmset
$4
user
$4
name
$2
tj
$5
email
$4
tobi
--------
*3
$4
hget
$4
user
$4
name
---------
*3
$4
hget
$4
user
$5
email
