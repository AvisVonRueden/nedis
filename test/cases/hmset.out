+OK
$2
tj
$4
tobi
